src/test/regress/sql/.gitignore

```diff
@@ -29,3 +29,7 @@
 /fdw_create.sql
 /fdw_data_types.sql
 /fdw_load.sql
+/columnar_chunk_filtering.sql
```

Comment:
Is this strictly necessary in this PR (or adding for convenience?)

Reply:
I don't think it is strictly necessary for this PR